homework/shelkovich_ilya/wrapperClasses/src/main/java/com/tutrit/java/quickstart/service/IntegerService.java

```diff
@@ -0,0 +1,34 @@
+package com.tutrit.java.quickstart.service;
+
+public class IntegerService {
+
+    public Integer byConstructor(Integer integer){
+        return integer;
+    }
```

Comment:
No, that is not how it works. 
`new Integer(5)`
that is by constructor

Reply:
I've did half of work. Did i get it right?